.vscode/extensions.json

```diff
@@ -0,0 +1,4 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=827846
```

Comment:
I guess these are autogenerated, but `//` comments aren't valid JSON. Not sure any comments are valid JSON.